src/app-layout/visual-refresh/layout.scss

```diff
@@ -26,6 +26,8 @@ explicitly set in script.
   #{custom-props.$contentGapLeft}: 0px;
   #{custom-props.$contentGapRight}: 0px;
   #{custom-props.$contentHeight}: calc(100vh - var(#{custom-props.$headerHeight}) - var(#{custom-props.$footerHeight}));
+  #{custom-props.$containerFirstGap}: 0px;
```

Comment:
I feel like there could be a better name for these properties but I'm not coming up with anything.